proto/buf.yaml

```diff
@@ -1,9 +1,11 @@
-# Generated by "buf config migrate-v1beta1". Edit as necessary, and
-# remove this comment when you're finished.
-#
-# This module represents the "proto" root found in
-# the previous configuration.
 version: v1
+name: buf.build/cosmos/ibc-go
```

Comment:
This must match the schema registry which @tac0turtle created! 

```suggestion
name: buf.build/cosmos/ibc
```

Reply:
If we move our proto files to cosmos/ibc. This will still be okay?

Reply:
yea, its not tied to a repo.

Reply:
I named ibc cause I wasn't sure if the proto files are specific to go or not. I can change if needed

Reply:
I think ibc should be fine. There's been talks about potentially moving protos to spec repo anyways